.github/workflows/build.yml

```diff
@@ -9,28 +9,64 @@ jobs:
   build:
     name: Build
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        java_version: ['11', '16']
     steps:
+      # Check out the project
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0  # Shallow clones should be disabled for a better relevancy of analysis
-      - name: Set up JDK 11
+
+      # Setup the version of Java
+      - name: Set up JDK ${{ matrix.java_version }}
         uses: actions/setup-java@v1
         with:
-          java-version: 11
+          java-version: ${{ matrix.java_version }}
+
+      # Cache all the things
       - name: Cache SonarCloud packages
-        uses: actions/cache@v1
+        uses: actions/cache@v2.1.4
```

Comment:
Update cache to 2.1.5 based on what we saw in Dropwizard...probably should do in kiwi too